### ISSUE TITLE
onPositionChanged constantly updated

### ISSUE THREAD

**Issue by jpapay** [fleaflet/flutter_map#877]
When moving the map, onPositionChanged callback is constantly updated. I need to call it only when the user lifts his finger from the screen.
I didn't find anything to achieve this behavior. Do you have an idea ?

**Comment by jpapay** [comment]
@johnpryan Hi, do you have an idea please ?

**Comment by kengu** [comment]
@jpapay I guess you should use `MapController.onTap` instead.

**Comment by jpapay** [comment]
@kengu Unfortunately, it doesn't work for my use case. When the user drags the map on his screen, I need to check the new bounds of the map displayed on his screen, but only when he lifts his finger from the screen.

Here's what I do actually :

```
onPositionChanged: (position, hasGesture) {
              print("NEW Bottom Left Corner: ${mapController.bounds.southWest}");
              print("NEW Up Right Corner: ${mapController.bounds.northEast}");
              print("NEW Zoom Level: ${mapController.zoom}");
            },
```
And for just a tiny drag, this is my output :
![image](https://user-images.githubusercontent.com/15807770/115849587-4d25cc00-a425-11eb-80cb-fd7fe70781ee.png)

**Comment by kengu** [comment]
@jpapay Okay, I see. `MapController.mapEventSink` publishes events that might be useful here. See [map_events.dart](https://github.com/fleaflet/flutter_map/blob/master/lib/src/gestures/map_events.dart) for list of all events published.

**Comment by jpapay** [comment]
Thank you, it seems to work like a charm !

**Comment by vinnytwice** [comment]
@kengu I assume that I'd have to set the listener in the initState method right? Also `.mapEventStream` throws `The getter 'mapEventStream' isn't defined for the type 'MapController'.` error.. I guess that it has to do with flutter_map version, I'm still using flutter_map: ^0.8.2 as I haven't upgraded to NullSafety yet... since which version is  MapController publishing events?
Many thanks.

**Comment by vinnytwice** [comment]
@jpapay @kengu guys can you guys please tell from from what version is MapEvent stream available? I'm on 10.0.1 but still not available to me..
